Userland/Libraries/LibWeb/DOM/Element.cpp

```diff
@@ -318,7 +318,7 @@ JS::GCPtr<Layout::Node> Element::create_layout_node_for_display_type(DOM::Docume
             return document.heap().allocate_without_realm<Layout::InlineNode>(document, element, move(style));
         if (display.is_flex_inside())
             return document.heap().allocate_without_realm<Layout::Box>(document, element, move(style));
-        dbgln_if(LIBWEB_CSS_DEBUG, "FIXME: Support display: {}", display.to_deprecated_string());
+        dbgln_if(LIBWEB_CSS_DEBUG, "FIXME: Support display: {}", display.to_string().release_value_but_fixme_should_propagate_errors());
```

Comment:
For `dbgln` I think `MUST` is okay

Reply:
Sure. I was a bit indecisive about it. These are both FIXMEs anyway so they'll go away eventually anyway.